app/controllers/editions/status_transitions_controller.rb

```diff
@@ -16,11 +16,10 @@ def create
 private
 
   def attempt_transition!(transition:)
-    case transition.to_sym
-    when :ready_for_2i
-      @edition.ready_for_2i!
+    if @edition.respond_to?(transition)
+      @edition.send("#{transition}!")
```

Comment:
I like this, but should we check if the edition responds to `"#{transition}!"` rather than just `transition`? I can see some potential weirdness where we could send a method that isn't a valid transition here.

With that in mind, maybe we should check it's a valid state too? Something like:

```
if @edition.available_transitions.include?(transition) && @edition.respond_to?("#{transition}!")
```